src/Aquifer.Data/Entities/BibleVersionWordGroupWordEntity.cs

```diff
@@ -2,7 +2,10 @@
 using Microsoft.EntityFrameworkCore;
 
 namespace Aquifer.Data.Entities;
+
 [PrimaryKey(nameof(BibleVersionWordGroupId), nameof(BibleVersionWordId))]
+[Index(nameof(BibleVersionWordId))]
+
```

Comment:
Nitpick since we were talking about consistency yesterday, we generally don't have an extra line between object and attribute.

Reply:
Agreed, this was an accident.  I'm going to turn off the VS "references" and code lens features so that I can see what is actual spacing and what is the IDE showing me "helpful" things that take up what looks like a line of code.  I had them off at my last job but figured I'd try them out again.